File: parfemologija-fe/src/app/model/review-model.ts
export interface FragranceReview {
  id: number;
  head: string;
  body: string;
  rating: number;
  createdBy: string;
  createdAt: string;
  fragranceId: number;
}

export class FragranceReviewCreate{
    constructor(
        public head: string,
        public body: string,
        public rating: number,
        public fragranceId: number
    ){}
}

File: parfemologija-be/src/main/java/ba/parfemologija/service/core/models/review/ReviewModel.java
package ba.parfemologija.service.core.models.review;

import io.swagger.v3.oas.annotations.media.Schema;
import lombok.Data;

import java.time.LocalDate;
import java.time.LocalDateTime;

@Data
@Schema(description = "All properties model of a review entity")
public class ReviewModel {
    @Schema(description = "ID of the review")
    private Long id;

    @Schema(description = "Headline of the review")
    private String head;

    @Schema(description = "Rating of the review")
    private Integer rating;

    @Schema(description = "Body of the review")
    private String body;

    @Schema(description = "Date of creation of the review")
    private LocalDate createdAt;

    @Schema(description = "Person who created the review")
    private String createdBy;

    @Schema(description = "Fragrance ID the review is associated with")
    private Long fragranceId;
}
